brigade-worker/prestart.js

```diff
@@ -2,19 +2,44 @@ const process = require("process")
 const fs = require("fs")
 const { execFileSync } = require("child_process")
 
-//checked out in repo
-const depsFile = "/vcs/brigade.json"
+const configFile = "/config.json";
```

Comment:
nit: why the naming difference between the file on disk (or in repo) and the file created by the worker?

Reply:
No particular reason; shall we change it back to `/brigade.json`?

Reply:
Updated to be `/brigade.json`